proposals/templates/proposals/table_and_text_with_header.html

```diff
@@ -0,0 +1,20 @@
+{% load i18n %}
+
+{% if page_break %}
+    <h2 style="page-break-before: always"> {% trans section_title %} </h2>
+{% elif section_title %}
+    <h2> {% trans section_title %} </h2>
+{% endif %}
```

Comment:
Correct me if I'm wrong, but isn't this implying a page break must have a section title?
Would it not be better to do something like this:
```django
{% if page_break %}
    <div style="page-break-before: always"></div>
{% endif %}
{% if section_title %}
    <h2> {% trans section_title %} </h2>
{% endif %}
```